website/content/api-docs/allocations.mdx

```diff
@@ -1039,5 +1039,22 @@ $ nomad operator api /v1/client/allocation/23f520cc-629a-46ff-395f-0661e7aa939e/
 }
 ```
 
+#### Field Reference
+
+- `ScheduledState` `(string)`: The task's current paused state. It can can have
+  one of the following values:
+
+  - `""` - The task is running. The only state returned for tasks with no
+    schedule.
+
+  - `force_run` - The task's scheduled has been overridden to run.
+
```

Comment:
this can also be `force_pause`:

```suggestion

- `force_pause` - The task's schedule has been overridden to pause.

```

as seen with:

```
$ n alloc pause -task=business -state=pause d0bbe4e2

$ n alloc pause -task=business -status d0bbe4e2
Pause state: "force_pause"

$ n operator api /v1/client/allocation/d0bbe4e2-f174-dca6-49f7-d7259a3fd639/pause?task=business
{"ScheduleState":"force_pause"}
```

that's the only gap by my eye, from the list here: https://github.com/hashicorp/nomad/blob/v1.9.0/nomad/structs/task_sched.go#L49-L55

Reply:
Yikes good catch! That's even the list I was working from, and I missed it. :grimacing: